sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -298,11 +299,6 @@ public UTF8String getUTF8String(int ordinal) {
   }
 
   @Override
-  public String getString(int ordinal) {
```

Comment:
it's the same with it's default implementation in `InternalRow`, so I think we don't need to reimplement it.